api/src/database/helpers/schema/dialects/cockroachdb.ts

```diff
@@ -11,6 +12,17 @@ export class SchemaHelperCockroachDb extends SchemaHelper {
 		await this.changeToTypeByCopy(table, column, type, options);
 	}
 
+	processField(field: Field): void {
```

Comment:
@licitdev do you recall the reasoning for https://github.com/directus/directus/commit/816c998291ecb1fceb7b7f867f94290896a3435f? I assume it's because knex-schema-inspector doesn't "recognize" this for CockroachDB and set the `has_auto_increment` flag, but also wondering if you have a better comment so wanted to tap you in here 😁

Reply:
Yeah it has been awhile, I recall it being along the lines of not correctly detecting `auto_increment` as the default value from the a string along the lines of `nextval(some_sequence_here)`.

Reply:
Pushed https://github.com/knex/knex-schema-inspector/pull/135. Added a task to update the `knex-schema-inspector` dependency.
Resolving this conversation for now.

Reply:
Awesome, thanks @licitdev! 🚀

Reply:
@licitdev Just to be sure: This means https://github.com/knex/knex-schema-inspector/pull/135 needs to be merged & released before merging this pull request?

Reply:
Yes, that is required.

Reply:
Okay, thanks for your confirmation! Sorry, just saw that you've even added this to the PR's description 🙈